sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregation/Aggregations.java

```diff
@@ -5,36 +5,16 @@
 
 package io.opentelemetry.sdk.metrics.aggregation;
 
-import io.opentelemetry.api.common.Labels;
-import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
-import io.opentelemetry.sdk.metrics.aggregator.AggregatorFactory;
-import io.opentelemetry.sdk.metrics.aggregator.CountAggregator;
-import io.opentelemetry.sdk.metrics.aggregator.DoubleLastValueAggregator;
-import io.opentelemetry.sdk.metrics.aggregator.DoubleMinMaxSumCountAggregator;
-import io.opentelemetry.sdk.metrics.aggregator.DoubleSumAggregator;
-import io.opentelemetry.sdk.metrics.aggregator.LongLastValueAggregator;
-import io.opentelemetry.sdk.metrics.aggregator.LongMinMaxSumCountAggregator;
-import io.opentelemetry.sdk.metrics.aggregator.LongSumAggregator;
-import io.opentelemetry.sdk.metrics.common.InstrumentDescriptor;
-import io.opentelemetry.sdk.metrics.common.InstrumentType;
-import io.opentelemetry.sdk.metrics.common.InstrumentValueType;
-import io.opentelemetry.sdk.metrics.data.MetricData;
-import io.opentelemetry.sdk.resources.Resource;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import javax.annotation.Nullable;
-import javax.annotation.concurrent.Immutable;
-
 public final class Aggregations {
+  private Aggregations() {}
 
   /**
    * Returns an {@code Aggregation} that calculates sum of recorded measurements.
    *
    * @return an {@code Aggregation} that calculates sum of recorded measurements.
    */
   public static Aggregation sum() {
```

Comment:
For followup guess we can move these into `Aggregation.java` itself

Reply:
Yes, but I try to limit the PR sizes :)